_includes/payment-transaction-states

```diff
@@ -0,0 +1,45 @@
+## Payment States And Transaction States
+
+Both payments and transactions find can be in several different states during
+the course of a purchase. You can find a short description of each state below.
+
+### Payments
+
+* `ready` means that the payment has been created successfully, and is now ready
+for further transaction operations, like `authorization`, `sale` or `abort`.
```

Comment:
Use the `rewrap` extension in VS Code to properly wrap the content in lists like this. It should ensure a hanging indent equal to the previous line.

```suggestion
   for further transaction operations, like `authorization`, `sale` or `abort`.
```